### PR TITLE
little changes to reflect as much as possible

### PR DESCRIPTION
**Pull request by tido-**
If someone comes by and is looking for a document management system, this person should understand right away what is available and what not.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Paperless
 |Travis|
 |Dependencies|
 
-Scan, index, and archive all of your paper documents
+Index and archive all of your scaned paper documents
```
Comment (Strubbl) [inline README.rst]: please change "scaned" to "scanned"

File: README.rst [hunk 2]
```diff
@@ -84,18 +84,18 @@ Similar Projects
 There's another project out there called `Mayan EDMS`_ that has a surprising
 amount of technical overlap with Paperless.  Also based on Django and using
 a consumer model with Tesseract and unpaper, Mayan EDMS is *much* more
-featureful and comes with a slick UI as well.  It may be that Paperless is
-better suited for low-resource environments (like a Rasberry Pi), but to be
-honest, this is just a guess as I haven't tested this myself.  One thing's
-for certain though, *Paperless* is a **much** better name.
+featureful and comes with a slick UI as well, but still in Python 2. It may be 
+that Paperless consumes less resources, but to be honest, this is just a guess
```
Comment (danielquinn) [inline README.rst]: Instead of "less", you should use "fewer" here.

File: README.rst [hunk 3]
```diff
@@ -23,17 +23,18 @@ it... because paper.  I wrote this to make my life easier.
 How it Works
 ============
 
-1. Buy a document scanner like `this one`_ (used by me) or `this other one`_
+1. Paperless does not run your scanner.
```
Comment (danielquinn) [inline README.rst]: This is a good point, and worth mentioning, but as it's not really a "step", it shouldn't be numbered.  Instead, maybe start this section with a paragraph:

> Paperless does not control your scanner, it only helps you deal with what your scanner produces:

**Comment by tido-** [comment]
Thank you both for your comments. I hope I changed it as you expected.

File: README.rst [hunk 4]
```diff
@@ -23,14 +23,16 @@ it... because paper.  I wrote this to make my life easier.
 How it Works
 ============
 
+Paperless does not control your scanner, it only helps you deal with what your 
+scanner produces
 1. Buy a document scanner like `this one`_ (used by me) or `this other one`_
```
Comment (danielquinn) [inline README.rst]: One last nitpick (sorry): you need a line break here, or you end up breaking the formatting.  Just look at the README on your branch in Github.  Once you fix this though, I'm happy to merge.

**Comment by tido-** [comment]
Hi Daniel, I am helplessly sorry.
I didn't know I could check this on my fork which is now obvious to me %)
In my Editor Kate everything looked fine, in nano, no error.
It needed an extra line in between, what I still don't understand.

So I had to do now a try and error with the gitHub system.
Thank you for being patient.

**Comment by Strubbl** [comment]
@tido- if you want, you could squash your many commits into one with git rebase, e.g. like described here http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by tido-** [comment]
@Strubbl  Thank you for this tip. This is only text (no code) and as a total rookie I rather leave it as it is, but I keep this in mind for the next time.

In my latest PR I have edited 3 documents - I didn't change the information given just tried to get around the rough edges for rookies here and there. Hope you like it.

**Comment by danielquinn** [comment]
Thanks for your contribution @tido-.  Merging now.

**Pull request merged**